utils/preprocessing.py

```diff
@@ -68,6 +68,9 @@ def preprocess_co2_emissions(df: pd.DataFrame) -> pd.DataFrame:
     # To work with tonnes of co2 it is necessary a conversion factor of 3.664
     df['co2'] = df['co2'] / 3.664
     df.rename(columns={'date': 'date', 'co2': 'co2_emissions'}, inplace=True)
+    df.to_pickle("./co2_emission")
+    # Filter dataset for a constant time period from 1965 onwards till 2019
+    df = df[(df['date'].dt.year > 1964) & (df['date'].dt.year < 2020)]
```

Comment:
Can we put de time_ini (1964) and time_end (2020) in the config file? So here we use variables? Just to get more control.

Reply:
Done